### PR TITLE
Fix orientation bug

### PR DESCRIPTION
**Pull request by ablaom**
#87

### CODE REVIEW

File: src/selection_heuristics.jl [hunk 1]
```diff
@@ -39,20 +39,20 @@ with measures that are neither `:loss` nor `:score` are reset to zero.
...
+        end
+    return NaiveSelection(weights)
+end
 
 function best(heuristic::NaiveSelection, history)
     first_entry = history[1]
     measures = first_entry.measure
     weights = measure_adjusted_weights(heuristic.weights, measures)
     measurements = [weights'*(h.measurement) for h in history]
     measure = first(history).measure[1]
```
Comment (OkonSamuel) [inline src/selection_heuristics.jl:55]: What's the `measure` variable here for?

File: src/selection_heuristics.jl [hunk 2]
```diff
@@ -39,20 +39,20 @@ with measures that are neither `:loss` nor `:score` are reset to zero.
 struct NaiveSelection <: SelectionHeuristic
     weights::Union{Nothing, Vector{Real}}
 end
-NaiveSelection(; weights=nothing) =
-    NaiveSelection(weights)
-
+function NaiveSelection(; weights=nothing)
```
Comment (OkonSamuel) [inline src/selection_heuristics.jl:42]: I feel you've changed the behaviour of `NaiveSelection` shouldn't the docstring be updated?

Comment (OkonSamuel) [reply src/selection_heuristics.jl:42]: Ok i see. The behaviour hasn't changed

Comment (ablaom) [reply src/selection_heuristics.jl:42]: No just added a check.

**Pull request merged**